sql/catalyst/src/main/scala/org/apache/spark/sql/errors/QueryCompilationErrors.scala

```diff
@@ -637,17 +637,21 @@ private[sql] object QueryCompilationErrors extends QueryErrorsBase {
   }
 
   def invalidFunctionArgumentsError(
-      name: String, expectedNum: String, actualNum: Int): Throwable = {
+      sqlExpr: Seq[Expression], name: String, expectedNum: String, actualNum: Int): Throwable = {
     new AnalysisException(
       errorClass = "WRONG_NUM_ARGS",
       messageParameters = Map(
+        "sqlExpr" -> sqlExpr.map(toSQLExpr(_)).mkString(","),
```

Comment:
This is not needed anymore. Could you remove it, please.

Reply:
Btw, seems like it's duplicated to https://github.com/apache/spark/pull/38707 ?

Can we just close this ticket ?